includes/class-activitypub.php

```diff
@@ -141,6 +141,10 @@ public static function render_activitypub_template( $template ) {
 		self::add_headers();
 
 		if ( ! is_activitypub_request() || ! should_negotiate_content() ) {
+			if ( \get_query_var( 'p' ) && Outbox::POST_TYPE === \get_post_type( \get_query_var( 'p' ) ) ) {
```

Comment:
```suggestion
			if ( Outbox::POST_TYPE === \get_query_var( 'post_type' ) ) {
```
Does this work? For custom post types, it should always define the `post_type` var, otherwise it defaults to `post`.

Reply:
Sadly not! That was my also my first idea ☺️

I would assume that it does not work because the post type is not public!?!